.github/workflows/vercel-preview-tumeet.yaml

```
@@ -1,74 +1,45 @@
-        run: vercel deploy --archive=tgz --prebuilt --token=${{ secrets.VERCEL_TOKEN }}
+        else
+          echo "tumeet_detected=false" >> $GITHUB_OUTPUT
+          echo "❌ apps-to-build.txt not found, skipping build"
+        fi
+
+  deploy-tumeet:
+    needs: check-changes
+    if: needs.check-changes.outputs.tumeet_detected == 'true'
+    uses: ./.github/workflows/build-app.yml
+    with:
+      vercel-project-id: ${{ secrets.VERCEL_TUMEET_PROJECT_ID }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)